### ISSUE TITLE
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table, in Widget Builder screen.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#885]
**Describe the bug**
scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**To Reproduce**

1. Go to Widget Builder  (http://localhost:9992/cm/app/?view=widgetbuilder&site=test&path=%2FSites%2Ftest%2Findex.html&pathType=page)
2. Create a new widget so that it is displayed in table, in Widget Builder home page.
3. Inspect element and you will find that scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.
4. Click on link "click to access package manager" given on right side of screen.
5. It will open a table in new tab in browser.
6. Inspect element and you will find that scope="col" is missing for th tag and scope = "row" is missing for very first td in each tr tag, in table.

**Expected behavior**
scope="col" for th tag and scope = "row" for very first td in each tr tag, in table, should be present.

**Additional context**
Navigate all the options available for similar issue and fix.
